docs/website/data/integrations.yaml

```diff
@@ -1356,6 +1356,30 @@ integrations:
     videos:
       - https://www.youtube.com/watch?v=RJkgmdjJn_w
 
+  topaz:
+    title: Topaz
+    description:  |
+      Topaz is an open source authorization service providing fine grained, real-time, policy based access control for applications and APIs. 
+      Topaz combines the best of OPA and Zanzibar in a single container that you can host as a sidecar or microservice.
```

Comment:
This sounds vague. What's the best of Zanzibar? What's the best of OPA? YMMV

Reply:
made this more precise